wpiutil/src/llvm/raw_ostream.cpp

```diff
@@ -845,12 +845,12 @@ raw_null_ostream::~raw_null_ostream() {
 #endif
 }
 
-void raw_null_ostream::write_impl(const char *Ptr, size_t Size) {
+void raw_null_ostream::write_impl(const char * /*Ptr*/, size_t /*Size*/) {
```

Comment:
I prefer static casting unused arguments to void to fix this warning. It keeps the function signatures clean.